### ISSUE TITLE
ImportError: numpy.core.multiarray failed to import

### ISSUE THREAD

**Issue by linqingfan** [bulletphysics/bullet3#3305]
Hi,
When I do import pybullet, it shows the error:
pybullet build time: Mar  4 2021 20:17:30
RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/usr/local/lib/python3.8/dist-packages/pybullet_envs/examples/__init__.py", line 10, in <module>
    from . import kukaGymEnvTest
  File "/usr/local/lib/python3.8/dist-packages/pybullet_envs/examples/kukaGymEnvTest.py", line 7, in <module>
    from pybullet_envs.bullet.kukaGymEnv import KukaGymEnv
  File "/usr/local/lib/python3.8/dist-packages/pybullet_envs/bullet/__init__.py", line 1, in <module>
    from pybullet_envs.bullet.cartpole_bullet import CartPoleBulletEnv
  File "/usr/local/lib/python3.8/dist-packages/pybullet_envs/bullet/cartpole_bullet.py", line 18, in <module>
    import pybullet as p2
ImportError: numpy.core.multiarray failed to import
May I know what could be the problem?

**Comment by lorepieri8** [comment]
@erwincoumans  
The Error arise in the pip version 3.0.9, while it's not there in 3.0.8.

This works without errors: 
`pip install pybullet==3.0.8
import pybullet as p`

**Comment by erwincoumans** [comment]
What OS and how was Python installed?

**Comment by lorepieri8** [comment]
Colab.

Create a new colab and run 

```
!pip install pybullet
import pybullet
```

to reproduce the error.

**Comment by erwincoumans** [comment]
pybullet 3.0.9 is compiled against latest numpy (1.20.1). It seems colab still uses 1.19.5
I'll upload a new package (3.0.10) against numpy 1.19.5. Alternatively, pip install numpy==1.20.1

**Comment by erwincoumans** [comment]
Should be fixed now with pybullet-3.1.0 compiled against numpy==1.19.5
~~~
!pip install pybullet --upgrade
import pybullet as p
~~~
